README.md

```diff
@@ -5,6 +5,19 @@ A command line tool to help with using Continuous Pipe as a remote development e
 This helps to set up Kubectl, create, build and destroy remote environments and keep files
 in sync with the local filesystem.
 
+## Prerequisites
+
+You will need the following:
+
+ * A Continuous Pipe hosted project with the GitHub integration set up
+ * The project checked out locally 
+ * The IP address, username and password to use for Kubenetes cluster
+ * rsync and fswatch installed locally
```

Comment:
FYI rsync should be installed by default on OS X, and fswatch can be installed with brew install fswatch. Does the script check for these dependencies at runtime?